server/mailet/mailets/src/main/java/org/apache/james/transport/util/SizeUtils.java

```diff
@@ -0,0 +1,35 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.transport.util;
+
+import com.jakewharton.byteunits.BinaryByteUnit;
+
+public class SizeUtils {
+
+    /**
+     * Return a human-readable version of the file size, where the input represents a specific number of bytes.
+     *
+     * @param size >= 0
+     */
+    public static String humanReadableSize(long size) {
+        return BinaryByteUnit.format(size, "#.#");
```

Comment:
Preconditions.checkArgument(size >= 0, "Size is required to be positive");

Reply:
I told you no need for it, this is tested by the underlaying library

Reply:
Commit removed anyway